aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/AWSCognitoAuthSession.java

```diff
@@ -110,8 +120,12 @@ public boolean equals(Object obj) {
     public String toString() {
         return new StringBuilder()
```

Comment:
For me, Android Studio generates a Lint about these `StringBuilder ` in `toString()`; it prefers just `"foo" + "bar"` style, as gets IDE-generated by default for `toString()`.

Reply:
Hmmm... that's weird since StringBuilder is more performant from what I understand. Maybe they don't think it's worth the extra dependency..? Anyway I'll update.

Reply:
I thought it was weird, too. To be fair, I didn't actually look deep enough into it, to tell if the IDE was being reasonable.